configure.ac

```diff
@@ -512,23 +512,29 @@ EXTRA_WARNINGS=""
 
 WARNLIST="
 	all
+	extra
+	unused
 	shadow
 	missing-prototypes
 	missing-declarations
+	suggest-attribute=noreturn
+	suggest-attribute=format
 	strict-prototypes
-	declaration-after-statement
 	pointer-arith
 	write-strings
 	cast-align
 	bad-function-cast
 	missing-format-attribute
+	float-equal
 	format=2
-	format-security
-	no-format-nonliteral
-	no-long-long
-	unsigned-char
-	gnu89-inline
-	no-strict-aliasing
+	format-signedness
+	shift-overflow=2
+	overlength-strings
+	redundent-decls
+	init-self
+	uninitialized
+	unknown-pragmas
+	no-unused-parameter
```

Comment:
libqb has many hundreds of function parameters that are unused. 
This prevents them being warned about.

Perhaps in the future, when those parameters are used or suppressed, this can be turned back on.